### PR TITLE
Telemetry: Implemented the basic logs collector and telemetry reporter in TestController 

### PR DESCRIPTION
**Pull request by mbivolan**
- Moved Get-SystemBasicLogs in the TestController so it can be reimplemented for each platform.
- The reported will look in the secrets file for the table name (under the <TableName> tag), if it cannot find it will default to "LISAv2Results" so it's backwards compatible with the current config.

### CODE REVIEW

File: TestControllers/AzureController.psm1 [hunk 1]
```diff
@@ -190,4 +190,64 @@ Class AzureController : TestController
 			}
 		}
 	}
+
+    [void] GetSystemBasicLogs($AllVMData, $User, $Password, $CurrentTestData, $CurrentTestResult, $enableTelemetry) {
```
Comment (lizzha) [inline TestControllers/AzureController.psm1]: Code duplicates for different platforms. My suggestion is to keep the Get-SystemBasicLogs in TestLogs.psm1, but move GetSQLQueryOfTelemetryData to TestProvider.psm1, then in TestController.psm1, call TestProvider.GetSQLQueryOfTelemetryData right after where Get-SystemBasicLogs is called

**Review by lizzha — CHANGES_REQUESTED** [review]

**Comment by juhlee-microsoft** [comment]
@mbivolan please add to Description

File: TestControllers/AzureController.psm1 (hunk 1 above)
Comment (lizzha) [reply TestControllers/AzureController.psm1]: or add a new method ReportTelemetryData to TestProvider.psm1, which get the sql query and upload to database.
This method can be overwritten by Providers of different platforms.
And it's called after Get-SystemBasicLogs, in TestController

Comment (mbivolan) [reply TestControllers/AzureController.psm1]: done

**Review by lizzha — APPROVED** [review]

**Comment by lizzha** [comment]
@mbivolan could you post the test result for Azure or HyperV?

**Pull request merged**